analyses/gene-set-enrichment-analysis/02-model-gsea.Rmd

```diff
@@ -0,0 +1,167 @@
+---
+title: "GSVA Score Modeling"
+author: "Stephanie J. Spielman for ALSF CCDL"
+date: '2020'
+output:
+  html_document:
+    df_print: paged
+    toc: yes
+  html_notebook:
+    df_print: paged
+    toc: yes
+    toc_float: yes
+params:
+  plot_ci: yes
+  is_ci: FALSE
+---
+
+### Purpose
+
+Models GSVA scores. **This code is NOT completed and should be regarded as a work _in progress_.**
```

Comment:
I see this note, but does this still mean you are ready for a first review?

Reply:
Ah shoot meant to remove that line; originally I was going to put heatmaps into this script also but had later changed my mind to do separate scripts for modeling and heatmap/other viz. So, yes, ready for first review!

Reply:
..That said, please wait at least 30 minutes to review because I am fixing a couple quick things now :) Some code was commented out that should not have been.

Reply:
Gimme a shout when you’re ready. 👍

Reply:
All yours!